lca-chimevc-stack/lambda_functions/chime_call_analytics_initialization/index.js

```diff
@@ -623,11 +626,11 @@ const handler = async function handler(event, context) {
         );
         return;
       }
-
-      callData = await getCallDataFromChimeEvents(event);
+      
+      callData = await getCallDataFromDdb(event.detail.callId);
```

Comment:
We should leave a comment that we load the copy of the callData so that we get the modified callid to be send to the KDS. The logic is not clear here.